<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<a target="_blank" href="C:\Users\vishal.shaha\workspace\PLRating_eApp_Version4\test-output\html\Mon_Sep_09_22_29_51_CDT_2019.png">Screenshot Link</a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href="C:\Users\vishal.shaha\workspace\PLRating_eApp_Version4\test-output\html\Mon_Sep_09_22_29_51_CDT_2019.png"><img src="C:\Users\vishal.shaha\workspace\PLRating_eApp_Version4\test-output\html\Mon_Sep_09_22_29_51_CDT_2019.png" height=200 width=200></a>]]>
    </line>
  </reporter-output>
  <suite name="Learning Testing" duration-ms="14452" started-at="2019-09-10T03:29:39Z" finished-at="2019-09-10T03:29:53Z">
    <groups>
    </groups>
    <test name="Home Page Test" duration-ms="14452" started-at="2019-09-10T03:29:39Z" finished-at="2019-09-10T03:29:53Z">
      <class name="Scratch.testScreenshot">
        <test-method status="PASS" signature="setUp()[pri:0, instance:Scratch.testScreenshot@139982de]" name="setUp" is-config="true" duration-ms="2220" started-at="2019-09-09T22:29:37Z" finished-at="2019-09-09T22:29:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:Scratch.testScreenshot@139982de]" name="beforeTest" is-config="true" duration-ms="11496" started-at="2019-09-09T22:29:39Z" finished-at="2019-09-09T22:29:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="clickNext()[pri:0, instance:Scratch.testScreenshot@139982de]" name="clickNext" duration-ms="705" started-at="2019-09-09T22:29:51Z" finished-at="2019-09-09T22:29:51Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Warning is wrong... expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Warning is wrong... expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at Scratch.testScreenshot.clickNext(testScreenshot.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<a target="_blank" href="C:\Users\vishal.shaha\workspace\PLRating_eApp_Version4\test-output\html\Mon_Sep_09_22_29_51_CDT_2019.png">Screenshot Link</a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href="C:\Users\vishal.shaha\workspace\PLRating_eApp_Version4\test-output\html\Mon_Sep_09_22_29_51_CDT_2019.png"><img src="C:\Users\vishal.shaha\workspace\PLRating_eApp_Version4\test-output\html\Mon_Sep_09_22_29_51_CDT_2019.png" height=200 width=200></a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- clickNext -->
        <test-method status="PASS" signature="AfterTest()[pri:0, instance:Scratch.testScreenshot@139982de]" name="AfterTest" is-config="true" duration-ms="2149" started-at="2019-09-09T22:29:51Z" finished-at="2019-09-09T22:29:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterTest -->
        <test-method status="PASS" signature="sendMail()[pri:0, instance:Scratch.testScreenshot@139982de]" name="sendMail" is-config="true" duration-ms="5593" started-at="2019-09-09T22:29:53Z" finished-at="2019-09-09T22:29:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sendMail -->
      </class> <!-- Scratch.testScreenshot -->
    </test> <!-- Home Page Test -->
  </suite> <!-- Learning Testing -->
</testng-results>
